jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/TCollectorUDPWriter.java

```diff
@@ -68,7 +68,7 @@ public TCollectorUDPWriter(
 			@JsonProperty("tagName") String tagName,
 			@JsonProperty("mergeTypeNamesTags") Boolean mergeTypeNamesTags,
 			@JsonProperty("metricNamingExpression") String metricNamingExpression,
-			@JsonProperty("addHostnameTag") Boolean addHostnameTag,
+			@JsonProperty("addHostnameTag") boolean addHostnameTag,
```

Comment:
All that class hierarchy uses `Boolean addHostnameTag` (with a capital `B`). This makes clear that this parameter is optional. Changing it to `boolean` is probably not what we want.